src/Sentry.Unity/SentryMonoBehaviour.cs

```diff
@@ -118,6 +118,7 @@ internal void OnApplicationFocus(bool hasFocus)
     internal partial class SentryMonoBehaviour
     {
         internal readonly MainThreadData MainThreadData = new();
+        internal Func<bool>? IsMainThread;
```

Comment:
Maybe we should instead change `MainThreadData` to an interface so it's mockable for tests, instead of pulling this one function?